sArena.xml

```diff
@@ -252,4 +253,4 @@
             <OnLoad method="OnLoad"/>
         </Scripts>
     </Frame>
-</Ui>
\ No newline at end of file
+</Ui>
```

Comment:
Can you make it so this like is not changed?

Reply:
> Can you make it so this line is not changed?

Hey, this should be fixed now.